test/performance/README.md

```diff
@@ -1,128 +1,168 @@
+
+    - 1 image stream.
+    - 4 deployments, each with two pod replicas (pause), mounting 4 secrets, 4 config maps, and
+      1 downward API volume each.
+    - 2 services, each pointing to the TCP/8080 and TCP/8443 ports of the first and second
+      deployments, respectively.
+    - 1 edge route pointing to the first service.
+    - 20 secrets containing a 2048-character random string.
+    - 10 config maps containing a 2048-character random string
+
+
+3. Configure the parameters of your experiment:  
 
    a) cd into the load configuration directory
 
-```bash
-$ cd  perf-tests/clusterloader2/testing/load
-```
-   b) create the `stress-test` namespace - you workload needs to be deployed into this exact namespace (temporary solution)
-
-```bash
-$ kubectl apply -f - <<EOF
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: stress-test
-  labels:
-    app.kubernetes.io/cluster: wec1
-EOF
-```
-
-   c) configure your sample workload: 
-   
-```bash
-$ vi test-workload.yaml
-```
+   ```bash
+   $ cd  $WORK_DIR
+   ```
   
-   d) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
    
-```bash
-$ vi performance-test-config.yaml
-``` 
+   ```bash
+   $ vi performance-test-config.yaml
+   ``` 
+
+   More specifically, configure the following parameters: 
+
+   - namespaces: number of namespaced (*default value: 1*)
+   - namespaceNamePrefix: name prefix for the namespace (*default value: "perf-exp"*)
+   - K8S_CLUSTER: set to `true` for Kubernetes clusters (*default value: "true"*)
+   - OPENSHIFT_CLUSTER: set to `true` for Kubernetes clusters (*default value: "false"*)
+   - tuningSet: workload generation function (*default value: "RandomizedLoad"*)
+
+   To learn more about the clusterloader2 load experiments configuration parameters, see the following: [clusterloader tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set).
+
+
+
 
-To learn more about the clusterloader2 load experiments configuration parameters, see the following: [clusterloader tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set)
+4. Run your first workload:
 
 
-3. Run your first workload:
+   Set ``--kubeconfig`` flag to the path of the kubeconfig file of your cluster (e.g., WDS kubeconfig) and use the newly added Kubestellar provider (`--provider=ks`)
 
+   ```bash
+   $ cd $WORK_DIR
+   $ go run cmd/clusterloader.go --testconfig=./testing/load/performance-test-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
+   ```
 
-Set ``--kubeconfig`` flag to the path of the kubeconfig file of your cluster (e.g., WDS kubeconfig) and use the newly added Kubestellar provider
+5. Clean up your enviroment:
```

Comment:
The script invoked below only affects Kubernetes API objects, it does not affect any files. This might be worth explicitly mentioning. Also, is it the right choice?

Reply:
Fixed. Clarify that is removing the workload Kubernetes API objects only. Yes, it's the right choice because we don't want to remove the cloned repo. If the user wants to do so, then, he/she must delete it manually.